doc/releases/changelog-dev.md

```diff
@@ -17,8 +17,17 @@
 
 <h3>Documentation</h3>
 
-<h3>Contributors</h3>
+<h3>Operator class refactor</h3>
+
+The Operator class has undergone a major refactor with the following changes:
+
+* The `diagonalizing_gates()` representation has been moved to the highest-level 
+  `Operator` class and is therefore available to all subclasses. A condition 
+  `qml.operation.defines_diagonalizing_gates` has been added, which can be used 
```

Comment:
I must admit, I am not so much a fan of this, it feels like a `capabilities` dictionary in another form 🤔

Reply:
To me this will be removed if we find a better queueing solution!